support-figma/extended-layout-plugin/src/localization-module.ts

```diff
@@ -314,3 +334,18 @@ async function clearNodeAsync(node: SceneNode) {
     }
   }
 }
+
+////////////////// Extras //////////////////
+function estimateCharlimit(text: string): number | "NONE" {
+  if (text.length > 40) return "NONE";
+  return Math.ceil(text.length * 0.3) * 5;
```

Comment:
How did you come up with this formula?

Reply:
go/android-localizable-strings#design-the-ui-layout-to-accommodate-longer-translations So I put estimate 1.5 of original length here.